README.rst

```diff
@@ -111,7 +111,7 @@ you need to install several packages:
 Create your first application
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Long story short, copy-paste this into the console:
+Long story short, copy-paste this into the console line-by-line:
```

Comment:
```suggestion
Long story short, enter these commands into the console one-by-one:
```

Documentation style guides (for example [Google's](https://developers.google.com/style/word-list#letter-c)) don't recommend using `copy-paste` wording